code/__HELPERS/icons.dm

```diff
@@ -1458,3 +1458,83 @@ GLOBAL_LIST_EMPTY(friendly_animal_types)
 	animate(src, pixel_x = pixel_x + shiftx, pixel_y = pixel_y + shifty, time = 0.2, loop = duration)
 	pixel_x = initialpixelx
 	pixel_y = initialpixely
+
+/**
+ * Returns a list that contains the leftmost, rightmost, bottom and top pixels in an atom's icon
+ */
+/atom/proc/get_bounding_box()
```

Comment:
This proc need replacement due to image proc performance. I am fine with instead just doing something dodgy and risking it falling off the edge of the icon. You could make the bug only placable on clothing if you wanted to reduce the risk of it being placed on a crowbar or any other thin icons. A solution needs to be found that doesn't use any byond icon procs